students/shibin_mathew/lesson9/mailroom.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import sys
+from collections import defaultdict
+
+
+class Donor:
+    def __init__(self, names, amount):
+        self.names = names
+        self.amount = amount
+
+    @property
+    def names(self):
+        return "{}".format(self._names)
+
+    @names.setter
+    def names(self, names):
+        self._names = names
+
+    @property
+    def amount(self):
+        return self._amount
+
+    @amount.setter
+    def amount(self, amount):
+        self._amount = amount
+
+
+class Collection:
+    def __init__(self, donors2=None):
+        self.donors2 = donors2
```

Comment:
what does `donors2` mean?

Reply:
passing in a donors list - I would assume in the industry you can access this from a database that holds the donor information.